src/apps/components/run.js

```diff
@@ -77,22 +86,27 @@ export class RunApp extends React.Component {
         this.props.dispatch(actions.newError(err));
     };
 
-    runView = () => {
+    runView = (options) => {
         let { dispatch } = this.props;
+        options = options || {};
 
         this.inputs.getValues()
         .then((values) => {
-            return this.view.run(this.props.bundle, values);
+            return this.view.run(this.props.bundle, values, options.debug);
```

Comment:
I'd make the last arg `options` in case we add more later. Both `bundle` and `values` are required-ish so we can keep them top level, but I think we can push the `debug` into a generic optional `options`.